packages/next-bundle-analyzer/index.js

```diff
@@ -0,0 +1,20 @@
+module.exports = (pluginOptions = { enabled: true }) => (nextConfig = {}) => {
```

Comment:
I believe this should read out like so:
```suggestion
module.exports = ({ enabled = true } = {}) => (nextConfig = {}) => {
```

This sets `enabled` to true when missing, _and_ supports the null object case.